### DISCUSSION TITLE
Script Executer example how to show live output

### DISCUSSION THREAD

**Discussion by hulleyrob** [zauberzeug/nicegui#2782] [Q&A] [answered]
### Question

This type of thing seems like it could be useful but what would be the best way to be able to show the output of a script in real time where there may be rows and rows of output over several minutes?

**Comment by falkoschindler** [comment, marked as answer]
Hi @hulleyrob,

The "python3 slow.py" button demonstrates how to show live updates while a script is running. Is this what you're looking for?

**Reply by hulleyrob** [reply]
Hi thanks for replying. 

Sorry I meant if I have a lot of output say a line every 10 seconds or so for many minutes. 

What should I be looking to use for the output as the dialog box is not suitable for this. 

Can this be done with a ui.list for example or are there any limitations on what can be used. 

Just asking before I go for the wrong road for what I'm trying to achieve. 

Thanks,

Rob

**Reply by falkoschindler** [reply]
I see. The [`ui.log` element](https://nicegui.io/documentation/log) is specifically designed for such things. But `ui.list` could work as well.

**Reply by hulleyrob** [reply]
Ok will have another go with the ui.log 

I just wasn't sure if the problems I was having trying to adopt the code to use list were some limitation I would not be able to get round or just with the code I was using thanks. 

Rob

**Reply by hulleyrob** [reply]
Yeah that's what I was looking for thank you have got it working with the ui.log
